configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,163 @@
+
+# Setting up a Raspberry Pi as a bridged wireless access point
+
+The Raspberry Pi can be used as a bridged wireless access point within an existing Ethernet network. This will extend the network to wireless computers and devices.
+
+If you wish to create a standalone wireless network, consider instead setting up a [routed access point.](./access-point-routed.md)
```

Comment:
Only a minor nitpick, but maybe the full-stop at the end of this sentence should be moved to after the `)`, rather than being before the `]` ?